packages/collector/src/agentConnection.js

```diff
@@ -267,9 +267,11 @@ exports.sendSpans = function sendSpans(spans, cb) {
     if (err && !maxContentErrorHasBeenLogged && err instanceof PayloadTooLargeError) {
       logLargeSpans(spans);
     } else if (err) {
-      logger.debug('Failed to send: %s', getSpanLengthInfo(spans));
+      const spanInfo = getSpanLengthInfo(spans);
+      logger.debug({ spanInfo }, 'Failed to send.');
```

Comment:
As discussed: Please use a standard log message `logger.debug(`Failed to send ${xxx}`).